vllm/config/__init__.py

```diff
@@ -4215,14 +4219,18 @@ def try_verify_and_update_config(self):
             return
 
         from vllm.model_executor.models.config import (
-            MODELS_CONFIG_MAP, HybridAttentionMambaModelConfig)
+            MODELS_CONFIG_MAP, HybridAttentionMambaModelConfig,
+            Mamba2ModelConfig)
         cls = MODELS_CONFIG_MAP.get(architecture, None)
         if cls is not None:
             cls.verify_and_update_config(self)
 
         if self.model_config.is_hybrid:
             HybridAttentionMambaModelConfig.verify_and_update_config(self)
 
+        if self.model_config.has_mamba2:
+            Mamba2ModelConfig.verify_and_update_config(self)
```

Comment:
what about maintaining a white list in HybridAttentionMambaModelConfig for models that should use v1? For example I think we can enable mamba1 and minimax soon. `has_mamba2` is only a very temporary way to determine v0 or v1.

Reply:
We could just add whitelist of models for now, but even once we support FCG for mamba1 and minimax, we still need a way to enable FCG by default for models that contain *only* mamba2 layers (e.g, `mistralai/Mamba-Codestral-7B-v0.1`). These models are not hybrid models so checking `is_hybrid` won't be sufficient.

Reply:
I've re-implemented it just adding a blacklist of models that aren't supported right now (it's only Lfm2 and Minimax).